### ISSUE TITLE
Numeric Field in Treetable or Datagrid - unit not aligned

### ISSUE THREAD

**Issue by derkoe** [porscheinformatik/clarity-addons#824]
When using a Numeric Field in a Treetable the unit symbol is not aligned correctly.

![image](https://user-images.githubusercontent.com/123199/87769450-f9df7700-c81d-11ea-8947-1748b55aafe2.png)

**Comment by derkoe** [comment]
The reason for this is the wrong `line-height` set by `.treetable .treetable-cell, .treetable .treetable-column`

**Comment by Wallabeng** [comment]
Love to see a contribution for this

**Comment by derkoe** [comment]
This is our workaround - I have no idea where to put this 
```
.treetable .treetable-cell, .treetable .treetable-column .unit {
  line-height: unset;
}
```

**Comment by d-m-s** [comment]
@Wallabeng can we integrate this fix/do we need another workaround for clr-addons?

**Comment by Wallabeng** [comment]
We shouldn't integrate this fix as it is, as the treetable should "know" about units.
Would be nice if we could create a StackBlitz for every issue, as I can give better suggestions then (for easier contributions).

**Comment by derkoe** [comment]
Here's a StackBlitz showing all issues with the Treetable https://stackblitz.com/edit/clarity-v3-light-theme-fa32z9

**Comment by derkoe** [comment]
This issue also occurs with a Clarity Datagrid - updated the StackBlitz

**Comment by Wallabeng** [comment]
The problematic css is the "position: absolute" on the ".numeric-input-wrapper .unit", that's why it loses the height information of its parent.
This should be solved with position:relative at least

**Comment by DaSmid** [comment]
Fixed with #2212
